pol-core/bscript/compiler/Compiler.cpp

```diff
@@ -32,10 +41,42 @@ void Compiler::write_included_filenames( const std::string& /*pathname*/ )
 {
 }
 
-bool Compiler::compile_file( const std::string& /*filename*/,
-                             const LegacyFunctionOrder* /*legacy_function_order*/ )
+bool Compiler::compile_file( const std::string& filename,
+                             const LegacyFunctionOrder* legacy_function_order )
 {
-  return false;
+  bool success;
```

Comment:
I'd initialize the variable already as false instead of setting it in the catch clause. But I'm aware it's a matter of taste.
```suggestion
  bool success = false;
```

Reply:
Fine with me!